package.json

```diff
@@ -3,8 +3,5 @@
   "version": "1.4.0",
   "peerDependencies": {
     "eslint": ">=3.14.1"
```

Comment:
do we want to suggest `v6.3.0` here?  that is where the typescript initializer stuff comes in?

also versions `v4.1.0` and `v5.8.0` have fixes to 'space-infix-ops' for typescript that i'm pretty sure are biting me right now with relation to typescript and styled-components (typescript templating).

i'm not sure what 'best practice' may be for minimum requirement, but i think in our world one of these may be a proper suggestion.